action.yml

```diff
@@ -5,26 +5,34 @@ branding:
   icon: circle
   color: red
 inputs:
-  new-image-name:
-    description: 'Name of the new image that will be created'
+  image:
+    description: 'The name (reference) of the image to build'
     required: true
   base-image:
-    description: 'Base image to use'
+    description: 'The base image to use to create a new container'
     required: false
+  dockerfiles:
+    description: 'The path of one or more Dockerfiles (default: ./Dockerfile)'
```

Comment:
specify that multiple dockerfiles should be separated by spaces

Reply:
I wrote the description similar to Docker action. Just mentioned is a list so users know they can add multiple values, then in the README we can specify better the format. BTW it can be changed for me.